src/ripple/nodestore/impl/Shard.cpp

```diff
@@ -392,7 +393,8 @@ Shard::isLegacy() const
 bool
 Shard::finalize(
     bool const writeSQLite,
-    boost::optional<uint256> const& expectedHash)
+    boost::optional<uint256> const& expectedHash,
+    const bool writeDeterministicShard)
```

Comment:
We usually don't put `const` on value params.

Reply:
Done.

Reply:
It looks like the `const` is still there.

Reply:
Done.